tests/TestCase/View/Widget/DateTimeWidgetTest.php

```diff
@@ -760,7 +761,7 @@ public function testRenderHourWidget12()
         $this->assertNotContains('value="0"', $result, 'No zero hour');
 
         $this->assertContains('<select name="date[meridian]">', $result);
-        $this->assertContains('<option value="pm" selected="selected">pm</option>', $result);
+        $this->assertContains('<option value="pm" selected="selected">PM</option>', $result);
```

Comment:
Since localization is used to represent numbers and text , ICU return lower case string for  am/pm marker [refrence](http://userguide.icu-project.org/formatparse/datetime)